src/main/java/module/series/SeriesPanel.java

```diff
@@ -28,15 +30,15 @@
 	private MatchDayPanel[] matchDayPanels;
 	private SeriesHistoryPanel seriesHistoryPanel;
 	private Model model;
-	private Map<Integer, Map<RatingsStatistics,Integer>> leagueStatistics;
+	//private Map<Integer, Map<RatingsStatistics, Model.StatisticsEntry>> leagueStatistics;	// team id -> (statistic key -> value)
```

Comment:
just remove commented out stuff from this class.

Reply:
removed